styles/base/classes.scss

```diff
@@ -0,0 +1,149 @@
+.wrapper {
```

Comment:
For the filename: would it be helpful to be more specific, instead of having classes.scss would it be better to rename this file to common.scss?